src/components/side-navigation/side-navigation-item.component.less

```diff
@@ -0,0 +1,77 @@
+ux-side-navigation-item {
+    position: relative;
+    display: block;
+    font-weight: 400;
+    font-size: 0.9375rem;
+
+    &.no-children {
+        &.active {
+            font-weight: 600;
+        }
+    }
+
+    &.collapse[style] {
+        height: auto !important;
+    }
+
+    &.active {
+        & > .menu-item {
+            & > .menu-item-title {
+                font-weight: bold;
+            }
+        }
+    }
+
+    &.fourth-level {

```

Comment:
Could you use classes with digits, e.g. `level-5`, which will make it easier to assign classes based on level.